Kernel/Devices/MajorNumberAllocation.h

```diff
@@ -9,6 +9,22 @@
 #include <AK/Array.h>
 #include <Kernel/API/DeviceFileTypes.h>
 
+// NOTE: PLEASE READ THIS NOTE CAREFULLY WHEN CHANGING THIS FILE!
```

Comment:
Let's not have this in two places. Either refer to the md file here, or put it only here.

Reply:
Sure. I will put it in the markdown file and refer to it.
A fix will be applied when sending a new revision.